src/routes/Tool/Tool.tsx

```diff
@@ -112,7 +110,7 @@ const ExploreOpportunities = () => {
         id="tool-your-opportunities-list"
```

Comment:
Not in this PR, but above: `<>{settingsOpen && <ToolSettings ref={firstSettingRef} />}</>` seems have unnecessary fragment tags `<>`.